src/thebelab.js

```diff
@@ -136,6 +137,7 @@ function renderCell(element, options) {
   // element should be a `<pre>` tag with some code in it
   let $cell = $("<div class='thebelab-cell'/>");
   let $element = $(element);
+  let $output = $element.next();
```

Comment:
This isn't necessarily present, we should check if the element has a correct selector, probably `data-output` class by default.

Reply:
If the next element is not present, then `$output` becomes an empty JQuery object, which results in `$output.data("output")` being `undefined`. This should not be a problem since no error is thrown and `$output.data("output")` still evaluates as `false` in the if statement.

Reply:
Rather I meant when the next element is present, but it isn't `data-output`, but rather any other DOM entry.

Reply:
In that case `$output.data("output")` would evaluate to `undefined` and thus `false`. I agree with you that using a selector is better and addressed this in the commit 1e626a2.